src/classes/ADDR_CannotDelete_TEST.cls

```diff
@@ -0,0 +1,451 @@
+        ];
+
+        System.assertEquals(TRUE, returnAddresses.isEmpty());
+
+        for (Database.DeleteResult result : results) {
+            System.assertEquals(TRUE, result.isSuccess());
+        }
+    }
+
+    /*********************************************************************************************************
+    * @description Test method for the hasChildRecords() method in ADDR_CannotDelete_TDTM to test that TRUE is
+    * returned when an Address record has child records of only type Contact.
+    */
+    @isTest
+    private static void testAddrHasChildContactRecordsIsTrue() {
+
+        UTIL_UnitTestData_TEST.ContactsWithAddrs addrsWithRelatedContact = UTIL_UnitTestData_TEST.createTestContactsAddrs(5);
+
+        List<Address__c> addrsWithChildContact = addrsWithRelatedContact.addrs; 
+
+        List<Address__c> returnAddresses = [
+            SELECT Id, 
+                (SELECT Id FROM Address__c.Contacts1__r)
+            FROM Address__c
+            WHERE Id IN :addrsWithChildContact
+        ];
+
+        ADDR_CannotDelete_TDTM addrCannotDeleteClass = new ADDR_CannotDelete_TDTM();
+
+        for (Address__c addr : returnAddresses) {
+            System.assertEquals(TRUE, addrCannotDeleteClass.hasChildRecords(addr));
+        }
+    }
+
+    /*********************************************************************************************************
+    * @description Test method for the hasChildRecords() method in ADDR_CannotDelete_TDTM to test that TRUE is
+    * returned when an Address record has child records of only type Account.
+    */
+    @isTest
+    private static void testAddrHasChildAccountRecordsIsTrue() {
+
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                            (Account_Processor__c = ADDR_CannotDelete_TEST.hhAccRecordTypeID));
+
+        UTIL_UnitTestData_TEST.AccsWithAddrs addrsWithRelatedAcc = UTIL_UnitTestData_TEST.createTestAccsAddrs(5, ADDR_CannotDelete_TEST.hhAccRecordTypeID);
+        
+        List<Address__c> addrsWithChildAcc = addrsWithRelatedAcc.addrs; 
+
+        List<Address__c> returnAddresses = [
+            SELECT Id, 
+                (SELECT Id FROM Address__c.Accounts__r)
+            FROM Address__c
+            WHERE Id IN :addrsWithChildAcc
+        ];
+
+        ADDR_CannotDelete_TDTM addrCannotDeleteClass = new ADDR_CannotDelete_TDTM();
+
+        for (Address__c addr : returnAddresses) {
+            System.assertEquals(TRUE, addrCannotDeleteClass.hasChildRecords(addr));
+        }
+    }
+
+    /*********************************************************************************************************
+    * @description Test method for the hasChildRecords() method in ADDR_CannotDelete_TDTM to test that TRUE is
+    * returned when an Address record has both Contact and Account child records.
+    */
+    @isTest
+    private static void testAddrHasChildRecordsIsTrue() {
```

Comment:
@m-kinkade Another stylistic change that might be worth considering in the future is naming the test explicitly like hasChildRecordsIsTrue. With the context of the test class, @isTest, and the comments provided, we can probably ease up on some of the abbreviations and go with more human-language test methods.